state/state.go

```diff
@@ -382,6 +383,34 @@ func (state *HelmState) DeleteReleases(helm helmexec.Interface, purge bool) []er
 	return nil
 }
 
+// TestReleases wrapper for executing helm test on the releases
+func (state *HelmState) TestReleases(helm helmexec.Interface, cleanup bool, timeout int) []error {
+	var wg sync.WaitGroup
+	errs := []error{}
+
+	for _, release := range state.Releases {
```

Comment:
Probably we want concurrency control in near future. We do that in e.g. [`ReleaseStatuses`](https://github.com/roboll/helmfile/pull/150/files#diff-f6be34387a141f652301c16cf7a5e36cR316).

But that's an another story and I'm ok with it isn't covered in this PR.
Just wanted to make it clear.